diff --git a/node_modules/@platejs/math/dist/index.js b/node_modules/@platejs/math/dist/index.js
index 9fbc88f..89e209e 100644
--- a/node_modules/@platejs/math/dist/index.js
+++ b/node_modules/@platejs/math/dist/index.js
@@ -68,7 +68,7 @@ var insertInlineEquation = (editor, texExpression, options) => {
 };
 
 // src/lib/BaseEquationPlugin.ts
-var import_katex_min = require("katex/dist/katex.min.css");
+
 var BaseEquationPlugin = (0, import_platejs3.createSlatePlugin)({
   key: import_platejs3.KEYS.equation,
   node: { isElement: true, isVoid: true }
diff --git a/node_modules/@platejs/math/dist/index.mjs b/node_modules/@platejs/math/dist/index.mjs
index ad2486d..f047036 100644
--- a/node_modules/@platejs/math/dist/index.mjs
+++ b/node_modules/@platejs/math/dist/index.mjs
@@ -28,7 +28,7 @@ var insertInlineEquation = (editor, texExpression, options) => {
 };
 
 // src/lib/BaseEquationPlugin.ts
-import "katex/dist/katex.min.css";
+
 var BaseEquationPlugin = createSlatePlugin({
   key: KEYS3.equation,
   node: { isElement: true, isVoid: true }
diff --git a/node_modules/@platejs/math/dist/react/index.js b/node_modules/@platejs/math/dist/react/index.js
index c28e750..960bbd6 100644
--- a/node_modules/@platejs/math/dist/react/index.js
+++ b/node_modules/@platejs/math/dist/react/index.js
@@ -70,7 +70,7 @@ var insertInlineEquation = (editor, texExpression, options) => {
 };
 
 // src/lib/BaseEquationPlugin.ts
-var import_katex_min = require("katex/dist/katex.min.css");
+
 var BaseEquationPlugin = (0, import_platejs3.createSlatePlugin)({
   key: import_platejs3.KEYS.equation,
   node: { isElement: true, isVoid: true }
diff --git a/node_modules/@platejs/math/dist/react/index.mjs b/node_modules/@platejs/math/dist/react/index.mjs
index 289d71f..e69ec16 100644
--- a/node_modules/@platejs/math/dist/react/index.mjs
+++ b/node_modules/@platejs/math/dist/react/index.mjs
@@ -31,7 +31,7 @@ var insertInlineEquation = (editor, texExpression, options) => {
 };
 
 // src/lib/BaseEquationPlugin.ts
-import "katex/dist/katex.min.css";
+
 var BaseEquationPlugin = createSlatePlugin({
   key: KEYS3.equation,
   node: { isElement: true, isVoid: true }
